specs/general-spec.js

```diff
@@ -0,0 +1,124 @@
+const { languages } = require('../dictionaries')
+
+describe('The Example App', () => {
+  context('basics', () => {
```

Comment:
This is a _huge_ nitpick, but all other contexts are capitalized, this should be too 😁  (also "global elements" on line 36) Fix this if you want, I'm approving it anyway 😉